include/irr/asset/IRenderpassIndependentPipeline.h

```diff
@@ -27,14 +28,33 @@ enum E_PRIMITIVE_TOPOLOGY
     EPT_PATCH_LIST = 10
 };
 
-struct SVertexInputParams
+enum E_VERTEX_INPUT_RATE
```

Comment:
make sure sizeof is 4, or use typed uint32_t enums

Reply:
resolved